src/plugins/intel_gpu/src/graph/graph_optimizer/prepare_primitive_fusing.cpp

```diff
@@ -742,6 +742,12 @@ void prepare_primitive_fusing::fuse_simple_primitives(program &p) {
                     // prelu fusion is not implemented in oneDNN3.1 (CVS-108233)
                     return;
                 }
+
+                if (activation_func == cldnn::activation_func::relu_negative_slope &&
+                    input.get_output_layout().batch() > 1 && input.is_type<convolution>()) {
```

Comment:
In case of dynamic layout `.batch()` will lead to exception
Additional check is required